tests/rptest/tests/prealloc_nodes.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2022 Vectorized, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+from rptest.tests.redpanda_test import RedpandaTest
+from ducktape.cluster.cluster_spec import ClusterSpec
+from ducktape.utils.util import wait_until
+from ducktape.tests.test import TestContext
+
+
+class PreallocNodesTest(RedpandaTest):
+    """
+    A mixin class that extends RedpandaTest to
```

Comment:
nit: this seems like a normal subclass rather than a mixin (in the pythonic way)

Reply:
Fair point. I'll change that in the followup PR